integrations/slack/src/handlers/lens.ts

```diff
@@ -1,8 +1,9 @@
 import { Logger } from '@gitbook/runtime';
 
-import { queryLens } from '../actions/queryLens';
+import { queryLens } from '../actions/queryLens'; // eslint-disable-line import/no-internal-modules
```

Comment:
To get rid of the disable line and in case there's a `actions/index.ts`.

import { queryLens } from '../actions';

Reply:
ah great. I was wondering why we had this.

Reply:
ah right, that causes a conflict between `actions/` and `actions.ts` which is why it reaches here. will try to resolve that